tests/test_attrs.py

```diff
@@ -0,0 +1,197 @@
+"""
+This file tests attrs support.
+
+See https://github.com/snok/flake8-type-checking/issues/77
+for discussion on the implementation.
+"""
+
+import textwrap
+
+import pytest
+
+from flake8_type_checking.codes import TC002
+from tests import _get_error
+
+
+@pytest.mark.parametrize(
+    'expected',
+    (
+        {'2:0 ' + TC002.format(module='decimal.Decimal')},
+        {'2:0 ' + TC002.format(module='decimal.Decimal')},
+    ),
+)
+def test_non_attrs_model(expected):
+    example = textwrap.dedent(
+        '''
+        from decimal import Decimal
+
+        class X:
+            x: Decimal
+        '''
+    )
+    assert _get_error(example, error_code_filter='TC001,TC002') == expected
+
+@pytest.mark.parametrize(
+    'imp, dec',
+    (
+        ["import attrs", "@attrs.define"],
+        ["import attr", "@attr.s(auto_attribs=True)"],
+        ["import attr", "@attr.define"],
+    ),
+)
+def test_attrs_model(imp, dec):
+    """
+    A class cannot be a attrs model if it doesn't have a base class,
+    so we should raise the same error here in both cases.
+    """
```

Comment:
Don't think this or the other 5 docstrings are up to date 🙂 Can you update them to explain the purpose of the test?

Reply:
Sorry, totally forgot about that.